sse2neon.h

```diff
@@ -3602,6 +3602,33 @@ FORCE_INLINE __m128i _mm_mulhi_epi16(__m128i a, __m128i b)
     return vreinterpretq_m128i_u16(r.val[1]);
 }
 
+// Multiply the packed unsigned 16-bit integers in a and b, producing
+// intermediate 32-bit integers, and store the high 16 bits of the intermediate
+// integers in dst.
+// https://software.intel.com/sites/landingpage/IntrinsicsGuide/#text=_mm_mulhi_epu16
+FORCE_INLINE __m128i _mm_mulhi_epu16(__m128i a, __m128i b)
+{
+    /* FIXME: issue with large values because of result saturation as
```

Comment:
Can you provide more precise comments which address the issue? That is, the range and/or condition to cause saturation,

Reply:
After checking out the document of both [Neon](https://developer.arm.com/architectures/instruction-sets/simd-isas/neon/intrinsics?search=vmull_u16) and [SSE](https://software.intel.com/sites/landingpage/IntrinsicsGuide/#text=_mm_mulhi_epu16), I think the saturation problem doesn't exist as 
[_mm_mulhi_epi16](https://github.com/DLTcollab/sse2neon/blob/master/sse2neon.h#L3724) does. Therefore, I have removed the comment